cocos/asset/assets/image-asset.jsb.ts

```diff
@@ -36,82 +38,54 @@ export type ImageAsset = JsbImageAsset;
 export const ImageAsset: typeof JsbImageAsset = jsb.ImageAsset;
 const jsbWindow = jsb.window;
 
-export interface IMemoryImageSource {
-    _data: ArrayBufferView | null;
-    _compressed: boolean;
-    width: number;
-    height: number;
-    format: number;
-    mipmapLevelDataSize?: number[];
-}
-
-export type ImageSource = HTMLCanvasElement | HTMLImageElement | IMemoryImageSource | ImageBitmap;
-
 const extnames = ['.png', '.jpg', '.jpeg', '.bmp', '.webp', '.pvr', '.pkm', '.astc'];
 
-function isImageBitmap (imageSource: any): boolean {
-    return !!(sys.hasFeature(sys.Feature.IMAGE_BITMAP) && imageSource instanceof ImageBitmap);
-}
-
-function isNativeImage (imageSource: ImageSource): imageSource is (HTMLImageElement | HTMLCanvasElement | ImageBitmap) {
-    if (ALIPAY || XIAOMI || BAIDU) {
-        // We're unable to grab the constructors of Alipay native image or canvas object.
-        return !('_data' in imageSource);
-    }
-    if (JSB && (imageSource as IMemoryImageSource)._compressed === true) {
-        return false;
-    }
-
-    return imageSource instanceof jsbWindow.HTMLImageElement || imageSource instanceof jsbWindow.HTMLCanvasElement || isImageBitmap(imageSource);
-}
-
 // TODO: we mark imageAssetProto as type of any, because here we have many dynamic injected property @dumganhar
 const imageAssetProto: any = ImageAsset.prototype;
 
-imageAssetProto._ctor = function (nativeAsset?: ImageSource) {
+imageAssetProto._ctor = function (nativeAsset?: ImageData | IMemoryImageSource | HTMLCanvasElement | HTMLImageElement | ImageBitmap) {
```

Comment:
Need to rename the parameter name `nativeAsset`.

Reply:
OK